seedfarmer/commands/_bootstrap_commands.py

```diff
@@ -0,0 +1,97 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License").
+#    You may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+import json
+import logging
+import os
+from typing import Any, Dict, List, Optional
+
+import yaml
+from jinja2 import Template
+
+from seedfarmer import CLI_ROOT
+from seedfarmer.utils import CfnSafeYamlLoader
+
+_logger: logging.Logger = logging.getLogger(__name__)
+
+
+def get_toolchain_template(
+    project_name: str,
+    principalARN: List[str],
```

Comment:
let's rename this so that it's obvious it's a list `principalARNs`

Reply:
Can do